files/initramfs-tools/union-mount.j2

```diff
@@ -6,17 +6,17 @@ case $1 in
 esac
 
 ## Mount the aufs file system: rw layer over squashfs
-mkdir -p ${rootmnt}/host/rw
-mount -n -o dirs=${rootmnt}/host/rw:${rootmnt}=ro -t aufs root-aufs ${rootmnt}
+mkdir -p ${rootmnt}/host/image-%%GIT_REVISION%%/rw
+mount -n -o dirs=${rootmnt}/host/image-%%GIT_REVISION%%/rw:${rootmnt}=ro -t aufs root-aufs ${rootmnt}
```

Comment:
>image-%%GIT_REVISION%% [](start = 33, length = 22)

One suggestion: could you also explore the possibility to dynamically get the path? Ideally the image content should not have the knowledge of its own revision/version.

Some mechanism I can think of, but not limited to:
1. kernel argument, provided by bootloader ONIE/Aboot/etc.
2. parse /proc/cmdline
3. parse a file with image version on the disk

Then we can remove the %GIT_REVISION% from build_debian.sh